### DISCUSSION TITLE
Release v2.0.1

### DISCUSSION THREAD

**Discussion by androidacy-user** [Fox2Code/FoxMagiskModuleManager#302] [General]
**Full Changelog**: [https://www.androidacy.com/magisk-module-manager-v2-0-1/](https://www.androidacy.com/magisk-module-manager-v2-0-1/?utm_source=github&utm_medium=web&utm_campaign=foxmmm-release-v2)

<hr /><em>This discussion was created from the release <a href='https://github.com/Fox2Code/FoxMagiskModuleManager/releases/tag/v2.0.1'>Release v2.0.1</a>.</em>

**Comment by amirism** [comment]
App doesn't launch after updating to v2.0.1. Just a blank, white screen.

Pixel 7 Pro, latest everything

**Reply by androidacy-user** [reply]
> App doesn't launch after updating to v2.0.1. Just a blank, white a screen.
> 
> Pixel 7 Pro, latest everything

If you read the blog post, there's a notice that users upgrading from older versions will need to clear data.

**Reply by schqual** [reply]
> > App doesn't launch after updating to v2.0.1. Just a blank, white a screen.
> > Pixel 7 Pro, latest everything
> 
> If you read the blog post, there's a notice that users upgrading from older versions will need to clear data.

Maybe better to write this here, at the release page? Your note in the bottom of post and not highlighted.

**Reply by androidacy-user** [reply]
> > > App doesn't launch after updating to v2.0.1. Just a blank, white a screen.
> > > Pixel 7 Pro, latest everything
> > 
> > If you read the blog post, there's a notice that users upgrading from older versions will need to clear data.
> 
> Maybe better to write this here, at the release page? Your note in the bottom of post and not highlighted.

Users are expected to read changelogs for changes and incompatibilities, especially for major version number bumps. I apologize for the inconveniences, but stuff isn't put into a changelog because we like hearing ourselves talk 😜

**Reply by mrkylegp** [reply]
I had the same issue after updating. Would just hang on the logo when opening. Uninstalled, reinstalled new version and that's worked. Saw the above post about clearing data. I'm also suggesting a note about clearing data be added to the release notes. The reality is most people aren't going to check the blog.

**Comment by Kay-Kostrowski** [comment]
I am stuck at the welcome menue. It is only possible to click cancel.
It seems the button "ready" is not even a button...

I have read the note about deinstalling the old version of MMM before and did it with 2 reboots after uninstalling and after installing new version.
Do I also need to uninstall all the modules before installing the new version of MMM?

**Reply by androidacy-user** [reply]
No, you need to scroll down to enable it. You may need to decrease your DPI if it's at an large value

**Reply by Kay-Kostrowski** [reply]
Thx for quick reply --> Problem Solved!
My screen resolution was to high... 😇


I am usually on 587dp for resolution in developer settings (left pic), that vanished the finish button somehow.
Reducing it down to 450dp brought back the finish button (right pic).
After welcome page, the rest of MMM just works fine with 587dp again.

Thx for help!
Never had an issue with resolution before and so it propable would have taken lots of time and brainmelt to find that simple solution by myself.

![AFFIX_20230427_064652.png](https://user-images.githubusercontent.com/84748960/234763650-6130a46f-0f77-4f46-9393-4a19b25402c6.png)

**Comment by Zharevnart** [comment]
![Screenshot_2023-04-27-10-26-46-57_115a3d4b87870cbbc4a93f5a8407539f](https://user-images.githubusercontent.com/87809229/234752298-da6cf0a1-39a4-4e61-93bc-3cbee107710d.jpg)
Fresh install
Magisk no pop-up to grant
But toast say already grant
And in menu say not granted

**Reply by androidacy-user** [reply]
Known issue for a minority of users but unknown what's causing it. That logic was untouched from v1.1 to v2

**Reply by happyBOY78** [reply]
Some problem on Realme C3, android 10

**Reply by ghost** [reply]
I have issue, any temp fix?

**Reply by lin0lin0** [reply]
same on Sony Xperia XZ3c

**Reply by ProfUgo55** [reply]
Same on Samsung Galaxy Tab A (SM-T580) running Android 8.1.0, build # M1AJQ.T580UES5CTL1.

Magisk log reads: "zygisk32: [com.samsung.android.securitylogagent] is on the denylist”

**Comment by jacko9000** [comment]
Will stick with v1.1.0 as I prefer the single list of modules with the progress bar whilst checking for updated modules, and the "Upgradable" list which then appears at the top if there are installed modules which can be updated.

**Reply by androidacy-user** [reply]
> Will stick with v1.1.0 as I prefer the single list of modules with the progress bar whilst checking for updated modules, and the "Upgradable" list which then appears at the top if there are installed modules which can be updated.

Both of those things are present in V2, the upgradable list is just under the online tab

**Reply by jacko9000** [reply]
> > Will stick with v1.1.0 as I prefer the single list of modules with the progress bar whilst checking for updated modules, and the "Upgradable" list which then appears at the top if there are installed modules which can be updated.
> 
> Both of those things are present in V2, the upgradable list is just under the online tab

No, they're not.  And no, it's just not.

V2 is not picking up modules which v1.1.0 lists as upgradable, and there is no progress bar in v2 when pulling down to refresh, and no "Upgradable" list.

I'm using Android 11 on a Poco X3 Pro.

**Comment by mrkylegp** [comment]
V2.0.1 broke everything for me. Redmagic 8 Pro.

Cleared the app data, tried to update some of my modules (revanced) and the app crashes with the following:

![Screenshot_2023-04-29-07-22-05-656](https://user-images.githubusercontent.com/52687659/235257335-82061663-23b9-4ed4-8abb-f8c291937cf5.jpg)


Went back to v1.1.0 and everything works again.

**Reply by Fox2Code** [reply]
Probs with how app data storage was changed

uninstall/reinsinstall or wiping app data should work.

**Reply by Handymann007** [reply]
Having the same problem on Galaxy S22 Ultra and Galaxy A22.  No matter which module I want to install.  :-(

**Reply by mrkylegp** [reply]
> Having the same problem on Galaxy S22 Ultra and Galaxy A22. No matter which module I want to install. :-(

Go back to v1.1.0 for now.

**Comment by PARAZIT44** [comment]
![Screenshot_2023-04-30-03-01-31-391_com fox2code mmm](https://user-images.githubusercontent.com/111461940/235328955-f038eeb6-47ea-4e1c-ab74-4d8921137da2.jpg)
Root access denied!
